cmd/blockimporter/state.go

```diff
@@ -224,8 +225,10 @@ func (state *State) processBlock(block types.Block, tx kv.RwTx) error {
 		return fmt.Errorf("some of the transactions were rejected")
 	}
 
-	if err := stateWriter.WriteHistory(); err != nil {
-		return fmt.Errorf("failed to write history: %v", err)
+	if saveHistoryData {
```

Comment:
usually, full nodes save a fixed amount of history (e.g. 128 blocks), maybe we want to do the same

Reply:
Erigon doesn't give an existing method to clear history for the old blocks. To I suggest to leave it as is for now